src/components/user-item.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {Text, View} from 'react-native';
+import {UserNode} from '../utils/models';
+
+const phoneMask = (value: string) => {
+  if (!value) {
+    return '';
+  }
```

Comment:
aqui value é obrigatório, mas vc valida se ele existe, n vale a pena indicar que pode ser opcional?

```suggestion
const phoneMask = (value?: string) => {
  if (!value) {
    return '';
  }
```